src/profile_manager.py

```diff
@@ -77,7 +77,7 @@ def write_xml(self, data):
             xml_string = []
 
         for dat in data:
-            xml_string.append("<record from=\"{}\" to=\"graphics/pictures/person/{}/portrait\"/>".format(dat[1]+"/"+dat[2], dat[0]))
+            xml_string.append("<record from=\"{}\" to=\"graphics/pictures/person/r-{}/portrait\"/>".format(dat[1]+"/"+dat[2], dat[0]))
```

Comment:
I imagine we'd want the tool to be backward compatible, and I think the `r-` stuff is only for FM24. Maybe worth having a conditional here for FM24? Not sure if we have a way to detect which version of FM we're working with... perhaps looking for 2024 in the graphics path?

Reply:
Cant really detect it automatically.

Reply:
Many players use custom location for their FM user folder.